save-backend/src/main/kotlin/com/saveourtool/save/backend/storage/AvatarStorage.kt

```diff
@@ -47,21 +62,21 @@ class AvatarStorage(configProperties: ConfigProperties) :
 
 /**
  * @property type
+ * @property folderUserName
  * @property imageName
- * @property filename
  */
 data class AvatarKey(
     val type: AvatarType,
+    val folderUserName: String,
```

Comment:
Maybe call it differently, because for `AvatarType.ORGNAIZATION` it won't be user name

Reply:
Ok. Renamed